apps/wallet/src/ui/app/pages/layout/index.tsx

```diff
@@ -23,17 +24,19 @@ function PageLayout({ forceFullscreen = false, children, className }: PageLayout
 	const isNavVisible = useAppSelector(getNavIsVisible);
 
 	return (
-		<Loading loading={guardLoading}>
-			<div
-				className={cl('w-popup-width h-popup-height', st.container, className, {
-					[st.navHidden]: !isNavVisible,
-				})}
-			>
-				{children}
-				<div id="overlay-portal-container"></div>
-				<div id="toaster-portal-container"></div>
-			</div>
-		</Loading>
+		<UnlockAccountProvider>
```

Comment:
is this needed for every page in the wallet or just some subset of pages? just thinking on whether or not this is the best place to put this provider haha

Reply:
Probably most of the pages where we need this use the Layout component but I think moving it to app/global level would make sense as well